lib/Controller/SettingsController.php

```diff
@@ -73,6 +74,7 @@ public function isDiscoveryEndpointValid($url) {
 		return $result;
 	}
 
+	#[PasswordConfirmationRequired]
```

Comment:
`OCP\AppFramework\Http\Attribute\PasswordConfirmationRequired` is only available in NC >= 27. You can use the old annotation instead:

```suggestion
	/**
	 * @PasswordConfirmationRequired
	 */
```

Reply:
> `OCP\AppFramework\Http\Attribute\PasswordConfirmationRequired` is only available in NC >= 27. You can use the old annotation instead:

Ah yes, got it, thank you! Not sure about the failing integration checks though, seems to be something with the php version?

Reply:
Yeah we can ignore those until we figure out how to fix them. The wrong version of Php is installed in there.